migration.go

```diff
@@ -94,6 +94,17 @@ set -e -v`)
 	return nil
 }
 
+func getShellScriptHeader(verbose bool) string {
+	var verboseFlag string
```

Comment:
its a matter of style, but i would write:

flag := "-v"
if verbose {
    flag = "-x"
}
return .....

Reply:
Completely agree, will change.